src/actinia_satellite_plugin/persistent_landsat_timeseries_creator.py

```diff
@@ -326,7 +329,7 @@ def _import_scenes(self):
 
             (
                 download_commands,
-                self.sentinel2_band_file_list,
+                self.landsat_band_file_list,
```

Comment:
I am not 100% sure that this has no unwanted behaviour as it might be used elsewhere, but I checked the related repositories as well and it seems, this is not further used, so fine for me.